nilearn/_utils/helpers.py

```diff
@@ -211,6 +211,16 @@ def compare_version(version_a, operator, version_b):
     return VERSION_OPERATORS[operator](parse(version_a), parse(version_b))
 
 
+def is_matplotlib_installed():
+    """Check if matplotlib is installed."""
+    try:
+        import matplotlib  # noqa: F401
+    except ImportError:
+        return False
+    else:
+        return True
```

Comment:
introducing this function: we have quite a few matplolib import check peppered in the code base: this could help refactor them